store/entity-store.html

```diff
@@ -133,21 +137,20 @@
 				const cacheKey = resolved.cacheKey;
 				const tokenValue = resolved.tokenValue;
 
-				const lowerCaseCacheKey = cacheKey.toLowerCase();
 				const lowerCaseEntityId = entityId.toLowerCase();
 
-				var entity = this._initContainer(this._store, entityId, cacheKey);
+				const entity = this._initContainer(this._store, entityId, cacheKey);
```

Comment:
Errr, I didn't think about it before, but how does `const` interact with IE11?

Reply:
https://caniuse.com/#feat=const

Supported.